Assets/Scripts/Player.cs

```diff
@@ -4,10 +4,13 @@
 
 public class Player : Entity
 {
+    public delegate void HealthChanged();
+    public static event HealthChanged healthChangedEvent;
```

Comment:
What is the reason for making this static?

Reply:
I was mostly just following a tutorial to throw something together that works for event listening/detection. New to C#, so not entirely sure how the static modifier interacts with events :)

Reply:
Ah ok. I'd think non-static would be more appropriate here. Doesn't really matter just wanted to point it out.